buildSrc/templates/oneDark.template.theme.json

```diff
@@ -1,9 +1,11 @@
-
 {
   "name": "One Dark",
   "dark": true,
   "author": "Mark Skelton",
   "editorScheme": "/themes/one_dark.xml",
+  "colors": {
+    "accentColor": "#568AF2"
+  },
```

Comment:
This is pretty neat

Reply:
Yeah! I started to do it with more colors but there was a lot. So I just stuck with this for the time being.